tf-training-tfjob/charts/values.yaml

```diff
@@ -26,11 +11,39 @@ resources:
     cpu: 19
     memory: 93604421089
 
+# Shorthands for setting both limits and requests of cpu/memory
 cpu: null
 memory: null
 
-experimentName: {{ NAUTA.ExperimentName }} 
-podCount: 1
+env:
+  - name: OMP_NUM_THREADS
```

Comment:
tf-training-tfjob is not a multinode training - so it doesn't use OpenMPI - thus those variables are superfluous here.